tests/test_schemas.py

```diff
@@ -0,0 +1,96 @@
+"""Container for tests of 2.03 schema compliance."""
+import glob
+import os
+import iati
+import lxml
+import pytest
+
+
+@pytest.fixture(scope='module')
+def activity_schema():
+    """Return the v2.03 IATI Activity Schema.
+
+    Todo:
+        Replace current monkeypatch of the default v2.02 schema with a call to `iati.schemas.ActivitySchema('iati-activities-schema.xsd')`
+        Requires resolution of issue: https://github.com/IATI/pyIATI/issues/234
+
+    """
+    schema = iati.default.activity_schema(version='2.02')
+    schema._schema_base_tree = lxml.etree.parse('iati-activities-schema.xsd')
+    return schema
+
+
+def get_filepaths_in_folder(directory_path):
+    """Return the full filepaths of all XML files within the input directory_path folder.
+
+    Args:
+        directory_path (str): A relative path to a directory to recursively find XML files within.
+
+    Returns:
+        list of str: Relative paths to all XML files that are contained within the input directory_path folder.
+
+    """
+    directory_glob_path = directory_path + '**'
+    all_files_in_directory = glob.glob(directory_glob_path, recursive=True)
+    return [path for path in all_files_in_directory if path.endswith(".xml")]
+
+
+def load_as_dataset(filepath):
+    """Load a filepath as an iati.Dataset object.
+
+    Args:
+        filepath (str): A relative path to an XML file.
+
+    Returns:
+        iati.Dataset: Representing the input XML file.
+
+    """
+    with open(filepath, 'r') as data:
+        xml_str = data.read()
+
+    return iati.Dataset(xml_str)
+
+
+@pytest.mark.parametrize('filepath', get_filepaths_in_folder('tests/activity-tests/should-pass/') +  # Legacy tests
+                         get_filepaths_in_folder('tests/should-pass/'))  # Tests in new format
+def test_pass_files(activity_schema, filepath):
+    """Check that all 'should-pass' test files are XML and pass Schema validation."""
+    # Load the dataset
```

Comment:
This comment seems very redundant.

(also present in other similar tests)

Reply:
https://github.com/IATI/IATI-Schemas/pull/367/commits/1d5f5b74aba79a9580387503e98d11d3619efbe7